ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml

```diff
@@ -127,6 +127,22 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-single-node
+- as: e2e-aws-gatewayapi
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_GATES: '[GatewayAPIEnabled=true]'
+      FEATURE_SET: CustomNoUpgrade
+    test:
+    - as: test
+      cli: latest
+      commands: TEST=TestGatewayAPI;make test-e2e
```

Comment:
`make TEST=TestGatewayAPI` is generally the better option. This approach is more direct and aligns with overriding make variables for the duration of a specific make invocation. It keeps the override scoped to the make process, avoiding potential side effects on the environment and other processes that do not need to be aware of the TEST variable.

Reply:
Good point.  It would be useful to explain why using an environment variable instead of a command-line argument is necessary, if it is.  

There is a difference in semantics between using an environment variable and using a command-line argument—quoting the manual:
```text
If a variable has been set with a command argument (*note Overriding
Variables: Overriding.), then ordinary assignments in the makefile are
ignored.  If you want to set the variable in the makefile even though it
was set with a command argument, you can use an 'override' directive,
```
It isn't obvious to me why we would want an assignment in the Makefile to override the `TEST=TestGatewayAPI` command-line argument.

Reply:
Updated, thanks for the feedback.